packages/shared/package.json

```diff
@@ -6,9 +6,13 @@
   "main": "./dist/index.js",
   "module": "./dist/index.mjs",
   "types": "./dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
   "scripts": {
     "dev": "npm run build -- --watch src",
     "build": "tsup src/index.ts --format cjs,esm --dts --clean",
+    "prepublishOnly": "npm run build",
```

Comment:
Let's remove this line. We're relying on turborepo to build the package and for script dependencies.

```suggestion
```